schematic/utils/cli_utils.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import inspect
+import logging
+
+from typing import Any, Mapping, Sequence, Union
+from functools import reduce
+
+from schematic import CONFIG
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+def query_dict(
+    dictionary: Mapping[Any, Any], keys: Sequence[Any]
+) -> Union[Any, None]:
+    """Access a nested value in a dictionary corresponding
+    to a series of keys.
+
+    Args:
+        dictionary: A dictionary containing anything.
+        keys: A sequence of values corresponding to keys
+            in `dictionary`
+
+    Returns:
+        The nested value corresponding to the given series
+        of keys, or `None` is such a value doesn't exist.
+    """
+
+    def extract(dictionary: Any, key: Any) -> Union[Any, None]:
+        """Get value associated with key, defaulting to None."""
+        if dictionary is None or not isinstance(dictionary, dict):
+            return None
+        return dictionary.get(key)
+
+    return reduce(extract, keys, dictionary)
+
+
+def fill_in_from_config(
+    arg_name: str,
+    arg_value: Any,
+    config_keys: Sequence[Any]
```

Comment:
Removed the `config` parameter that was previously being passed to `fill_in_from_config()`. Now it just uses the global `CONFIG` object.